docs/api-conventions.md

```diff
@@ -232,67 +232,8 @@ TODO: Plugins, extensions, nested kinds, headers
 HTTP Status codes
 -----------------
 
-The following HTTP status codes may be returned by the API.
-
-#### Success codes
-
-* `200 StatusOK`
-  * Indicates that the request completed successfully.
-* `201 StatusCreated`
-  * Indicates that the request to create kind completed successfully.
-* `204 StatusNoContent`
-  * Indicates that the request completed successfully, and the response contains no body.
-  * Returned in response to HTTP OPTIONS requests.
-
-#### Error codes
-* `307 StatusTemporaryRedirect`
```

Comment:
There is no status section for it.  I don't think we should be redefining the http spec in text in general - in this case I think some of these should be in swagger, not here.


> On Mar 27, 2015, at 3:13 PM, Nikhil Jindal <notifications@github.com> wrote:
> 
> In docs/api-conventions.md:
> 
> > @@ -232,67 +232,8 @@ TODO: Plugins, extensions, nested kinds, headers
> >  HTTP Status codes
> >  -----------------
> >  
> > -The following HTTP status codes may be returned by the API.
> > -
> > -#### Success codes
> > -
> > -* `200 StatusOK`
> > -  * Indicates that the request completed successfully.
> > -* `201 StatusCreated`
> > -  * Indicates that the request to create kind completed successfully.
> > -* `204 StatusNoContent`
> > -  * Indicates that the request completed successfully, and the response contains no body.
> > -  * Returned in response to HTTP OPTIONS requests.
> > -
> > -#### Error codes
> > -* `307 StatusTemporaryRedirect`
> The section below does not talk about this and the success codes.
> 
> —
> Reply to this email directly or view it on GitHub.
>